docs/rules.md

```diff
@@ -1,341 +1,119 @@
-| :white_check_mark: | warning | Web extension | Temporary IDs can cause issues with storage.sync | | | null | STORAGE_SYNC |
-| :white_check_mark: | warning | Web extension | Temporary IDs can cause issues with identity.getRedirectURL | | | null | IDENTITY_GETREDIRECTURL |
-| :white_check_mark: | warning | Web extension | When default_locale is specified a matching messages.json must exist | | | null | NO_MESSAGES_FILE |
-| :white_check_mark: | warning | Web extension | When _locales directory exists, default_locale must exist | | | null | NO_DEFAULT_LOCALE |
-| :white_check_mark: | warning | Web extension | | | | | UNSAFE_VAR_ASSIGNMENT |
-| :white_check_mark: | warning | Web extension | Unsupported or unknown browser API | | | null | UNSUPPORTED_API |
-| :white_check_mark: | warning | Web extension | | | | | DANGEROUS_EVAL |
-| :white_check_mark: | warning | Web extension | strict_max_version not required | | | null | STRICT_MAX_VERSION |
-| :white_check_mark: | error   | Web extension | Temporary error on content script `matches` | | | | MANIFEST_INVALID_CONTENT |
-| :white_check_mark: | error   | Web extension | Content script file could not be found | | | | CONTENT_SCRIPT_NOT_FOUND |
-| :white_check_mark: | error   | Web extension | Content script file name should not be empty | | | | CONTENT_SCRIPT_EMPTY |
-| :white_check_mark: | error   | Web extension | Translation string is missing the message property | | | | NO_MESSAGE |
-| :white_check_mark: | warning | Web extension | String name is reserved for a predefined message | | | | PREDEFINED_MESSAGE_NAME |
-| :white_check_mark: | error   | Web extension | String name contains invalid characters | | | | INVALID_MESSAGE_NAME |
-| :white_check_mark: | warning | Web extension | Placeholder for message is not defined | | | | MISSING_PLACEHOLDER |
-| :white_check_mark: | error   | Web extension | Placeholder name contains invalid characters | | | | INVALID_PLACEHOLDER_NAME |
-| :white_check_mark: | error   | Web extension | Placeholder is missing the content property | | | | NO_PLACEHOLDER_CONTENT |
+| Message code                | Type      | Description                         |
+| ----------------------------|-----------|-------------------------------------|
+| `TYPE_NO_MANIFEST_JSON`     | notice    | Add-on missing manifest_json for type detection |
+
+## Language packs
+
+| Message code                | Type      | Description                         |
+| ----------------------------|-----------|-------------------------------------|
+| FLUENT_INVALID              | warning   | Invalid fluent template file        |
+
+## Web Extensions / manifest.json
+
+| Message code                | Type          | Description                         |
+| ----------------------------|---------------|-------------------------------------|
+| `WRONG_ICON_EXTENSION`      | error         | Icons must have valid extension. |
```

Comment:
Just a thought, but it might be nice to split the errors and warnings separately.

Reply:
Hmm, or sort them? I wasn't really happy about it either.